lib/kb_fasttree/kb_fasttreeImpl.py

```diff
@@ -50,63 +26,28 @@ class kb_fasttree:
     # the latter method is running.
     ######################################### noqa
     VERSION = "1.1.0"
-    GIT_URL = "https://github.com/kbaseapps/kb_fasttree"
-    GIT_COMMIT_HASH = "b967ee863c008d6b131ffb70569e536dc863f127"
-
-    #BEGIN_CLASS_HEADER
-    FASTTREE_bin = '/kb/module/FastTree/bin/FastTree'
-
-    def now_ISO(self):
-        now_timestamp = datetime.now()
-        now_secs_from_epoch = (now_timestamp - datetime(1970,1,1)).total_seconds()
-        now_timestamp_in_iso = datetime.fromtimestamp(int(now_secs_from_epoch)).strftime('%Y-%m-%d_%T')
-        return now_timestamp_in_iso
-
-    def log(self, target, message):
-        message = '['+self.now_ISO()+'] '+message
-        if target is not None:
-            target.append(message)
-        print(message)
-        sys.stdout.flush()
```

Comment:
both these functions have been transferred over to fasttree.py

I simplified `now_ISO` to just record a timestamp for `now()`